src/org/opensolaris/opengrok/configuration/Groups.java

```diff
@@ -329,9 +329,10 @@ private static void deleteGroup(Set<Group> groups, String groupname) {
     private static boolean treeTraverseGroups(Set<Group> groups, Walker f) {
```

Comment:
even though this is private method, javadoc would be nice (not only saying how/where this is used but also what the walker does etc.)